stix_shifter_modules/qradar/stix_translation/query_constructor.py

```diff
@@ -209,6 +209,10 @@ def _parse_comparison_expression(self, expression, qualifier=None):
         # Resolve the comparison symbol to use in the query string (usually just ':')
         comparator = self._lookup_comparison_operator(self, expression.comparator)
 
+        # Special case where we want the risk finding
+        if stix_object == 'x-risk-finding' and stix_field == 'name' and expression.value == "*":
+            return "devicetype = 18"
```

Comment:
Not sure why we need this special case. can't they just stick in devicetype in stix-pattern `[x-com-ibm-ariel:devicetype = 18]` along with `x-risk-finding` pattern?

Reply:
also allowing special cases like this may increase our code complexity. I know there are few more QRadar internal device type like this.

Reply:
It's because the AQL query for findings uses that specific device type id. That's why I'm using a pretty specific if statement. You can see in the unit test the specific pattern that will be used to query QRadar findings.

Reply:
You can still search for a another cre name if needed. This is specific to doing a very specific pattern.